examples/micro_batching/micro_batching_handler.py

```diff
@@ -0,0 +1,34 @@
+import json
+import logging
+from pathlib import Path
+
+from micro_batching import MicroBatching
+
+from ts.torch_handler.image_classifier import ImageClassifier
+
+logger = logging.getLogger(__name__)
+
+
+class MicroBatchingHandler(ImageClassifier):
+    def __init__(self):
+        mb_handle = MicroBatching(self)
+        self.handle = mb_handle
+
+    def initialize(self, ctx):
+        super().initialize(ctx)
+
+        config_file = Path(ctx.system_properties["model_dir"], "micro_batching.json")
+        if config_file.exists():
+            try:
+                with open(config_file) as f:
+                    config = json.load(f)
+
+                self.handle.parallelism = config["parallelism"]
+                self.handle.micro_batch_size = config["micro_batch_size"]
```

Comment:
users may be thought "parallelism" and "micro_batch_size" are related to large model parallel. Can we have a good naming to differentiate?

Reply:
Sure, can just call them mb_size and mb_parallelism

Reply:
Went with ["micro_batching"]["parallelism"], that should be sufficient for users to differentiate.